component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/EmailOtpService.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
please update the license header

Reply:
fixed in 99ef19caaf9cd3d8a29b38a5d0c58726f2c97e49